modules/monitoring-creating-alerting-rules.adoc

```diff
@@ -39,6 +43,15 @@ spec:
 ----
 +
 This configuration creates an alerting rule named `example-alert`, which fires an alert when the `version` metric exposed by the sample service becomes `0`.
++
+[IMPORTANT]
+====
+For every namespace, you can use metrics of that namespace and cluster metrics, but not metrics of another namespace.
+
+For example, an alerting rule for `ns1` can have metrics from `ns1` and cluster metrics, such as the CPU and memory metrics. However, it cannot have metrics from `ns2`.
+
+Additionally, you cannot create alerting rules for the `openshift-*` namespaces, because they are immutable.
```

Comment:
Maybe a better wording would be:
> Additionally, you cannot create alerting rules for core openshift namespaces (i.e. `openshift-*`).

Reply:
I would even say something like to explain the reason why:
> Out of the box with OpenShift we provide core cluster alerts for OpenShift namespaces, therefor you do not need to create alerting rules based on `openshift-*` namespaces.